CMakeLists.txt

```diff
@@ -605,6 +605,10 @@ option(SWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY
   "Enable build of the Swift concurrency module"
   FALSE)
 
+option(SWIFT_ENABLE_EXPERIMENTAL_CXX_INTEROP
+  "Enable experimental C++ interop modules"
+  FALSE)
```

Comment:
Ah, food point.  We should update the presets - that seems like the better approach.  Even the concurrency library is handled the same way.

Reply:
Digging further, it seems that for concurrency we force the default to be be `TRUE` for Darwin and Linux through a [build-script flag](https://github.com/apple/swift/blob/main/utils/build_swift/build_swift/driver_arguments.py#L1324)

Do you think it would be a good idea to match that pattern here?